source/includes/fundamentals/code-snippets/srv.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+// Connection URI
+const uri = "mongodb+srv://sample-hostname:27017/?poolSize=20&writeConcern=majority"
```

Comment:
This is an invalid URI.
- SRV URIs must not include a port number.
- `poolSize` is not a valid URI option.
- `writeConcern` is not a valid URI option.

```suggestion
const uri = "mongodb+srv://sample-hostname/?maxPoolSize=20&w=majority"
```

Reply:
Next commit doesn't include string fix, still sorting that out/coordinating fixes for other drivers.